[2025-03-14T18:52:11.045+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-14T18:52:11.206+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: open_weather_etl.transform scheduled__2025-03-14T18:45:00+00:00 [queued]>
[2025-03-14T18:52:11.210+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: open_weather_etl.transform scheduled__2025-03-14T18:45:00+00:00 [queued]>
[2025-03-14T18:52:11.210+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 2
[2025-03-14T18:52:11.214+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): transform> on 2025-03-14 18:45:00+00:00
[2025-03-14T18:52:11.219+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=266) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-14T18:52:11.220+0000] {standard_task_runner.py:72} INFO - Started process 267 to run task
[2025-03-14T18:52:11.221+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'open_weather_etl', 'transform', 'scheduled__2025-03-14T18:45:00+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/open_weather_dag.py', '--cfg-path', '/tmp/tmpz0tprvap']
[2025-03-14T18:52:11.221+0000] {standard_task_runner.py:105} INFO - Job 108: Subtask transform
[2025-03-14T18:52:11.245+0000] {task_command.py:467} INFO - Running <TaskInstance: open_weather_etl.transform scheduled__2025-03-14T18:45:00+00:00 [running]> on host a8e3300004fc
[2025-03-14T18:52:11.283+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='lherrera5034@gmail.com' AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='open_weather_etl' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-03-14T18:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-14T18:45:00+00:00'
[2025-03-14T18:52:11.284+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-14T18:52:11.297+0000] {logging_mixin.py:190} INFO - Data trasnformed
[2025-03-14T18:52:11.302+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-14T18:52:11.306+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-14T18:52:11.306+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=open_weather_etl, task_id=transform, run_id=scheduled__2025-03-14T18:45:00+00:00, execution_date=20250314T184500, start_date=20250314T185211, end_date=20250314T185211
[2025-03-14T18:52:11.320+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-14T18:52:11.334+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-14T18:52:11.335+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
